layermanager/layermanager.go

```diff
@@ -85,6 +89,7 @@ type LayerManager struct {
 	layerAllocator    spaceallocator.Allocator
 	downloadAllocator spaceallocator.Allocator
 	extractAllocator  spaceallocator.Allocator
+	ttlStopChannel    chan bool
```

Comment:
All comments from service manager are applicapble for this commit as well as add GetCachedLayers storage API.

Reply:
Done